defaults/main.yaml

```diff
@@ -1,4 +1,10 @@
 ---
+cloud_vpn_install_pip_requirements: false
+cloud_vpn_install_providers: false
+cloud_vpn_install_provisioners: false
+cloud_vpn_providers_file: "{{ role_path }}/providers_from_devel.yaml"
+cloud_vpn_provisioners_file: "{{ role_path }}/provisioners_from_devel.yaml"
```

Comment:
Shouldn't this default to installing from the galaxy in the released version of the role as the devel branch for provider roles might unintentionally break some functionality when using a released version of `cloud_vpn` role.

Reply:
+1